### PR TITLE
Add tab completion

### LINKED ISSUE #119: Tab completion for `t2 ...`
See: https://github.com/gruntjs/grunt-cli

### PR DESCRIPTION
**Pull request by chrisbodhi**
Add tab completion for Bash and Zsh users. Update README with instructions on getting going with tab completion.

:bookmark_tabs:

Closes #119.

### CODE REVIEW

**Comment by rwaldron** [comment]
I'll give this a thorough review first thing tomorrow!

File: _t2 [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+  if (( CURRENT == 2 ));
+    then
+    _describe -t commands 'Commands' commands
+  fi
+
+  return 0
+}
+
+_t2
```
Comment (LinusU) [inline _t2]: [There should be a new line at the end of the file](http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)

**Comment by rwaldron** [comment]
@chrisbodhi can you fix the item that @LinusU mentioned and be sure to sign the CLA: http://dojofoundation.org/about/claForm 

Once you've done both things, just ping me here :)

File: _t2 (hunk 1 above)
Comment (chrisbodhi) [inline _t2]: Got it!

**Comment by chrisbodhi** [comment]
@rwaldron Added the newline & I already signed the CLA when I submitted another PR a few months ago. Should be good to go, but let me know if there's anything else you need from me!

**Comment by rwaldron** [comment]
Thanks for the reminder—this is good to go!

**Comment by johnnyman727** [comment]
:clap: Thanks for contributing again @chrisbodhi! I'm excited to use this.

**Pull request merged**